procedures/release-duty/index.rst

```diff
@@ -65,25 +65,21 @@ Email
 
 As ReleaseDuty you need to *subscribe* to certain mailing lists:
 
- * All types of sign-offs and approvals should go to `release signoff mailing list <https://mail.mozilla.org/listinfo/release-signoff>`__
- * All discussion topics should go to `release drivers mailing list <https://mail.mozilla.org/listinfo/release-drivers>`__
-
-These other mailing lists will likely have useful discussions and information:
+These mailing lists will likely have useful discussions and information:
```

Comment:
Nit: we should probably rename this to something like: "Please join the following groups to stay on top of the communication regarding the releases." or alike.